packages/data-stores/src/plans/resolvers.ts

```diff
@@ -95,11 +98,59 @@ function processFeatures( features: Feature[] ) {
 			name: feature.name,
 			description: feature.description,
 			type: feature.type ?? 'checkbox',
+			requiresAnnuallyBilledPlan:
+				FEATURE_IDS_THAT_REQUIRE_ANNUALLY_BILLED_PLAN.indexOf( feature.id ) > -1,
 		};
 		return features;
 	}, {} as Record< string, PlanFeature > );
 }
 
+function doesFeatureRequireAnnuallyBilledPlan(
+	featureName: string,
+	allFeaturesData: Record< string, PlanFeature >
+): boolean {
+	const matchedFeatureId = Object.keys( allFeaturesData ).find(
+		( featureId ) => allFeaturesData[ featureId ].name === featureName
+	);
+
+	if ( matchedFeatureId ) {
+		return allFeaturesData[ matchedFeatureId ].requiresAnnuallyBilledPlan;
+	}
+
+	return false;
+}
+
+function processPlanFeatures(
+	planData: APIPlanDetail,
+	allFeaturesData: Record< string, PlanFeature >
+): PlanSimplifiedFeature[] {
+	const features: PlanSimplifiedFeature[] = planData.highlighted_features.map(
+		( featureName ) => ( {
+			name: featureName,
+			requiresAnnuallyBilledPlan: doesFeatureRequireAnnuallyBilledPlan(
+				featureName,
+				allFeaturesData
+			),
+		} )
+	);
+
+	// Features requiring an annually billed plan should be first in the array.
+	features.sort( ( a, b ) => {
+		if ( a.requiresAnnuallyBilledPlan === b.requiresAnnuallyBilledPlan ) {
+			// Either they both require an annually billed plan, or they both don't
+			return 0;
+		} else if ( a.requiresAnnuallyBilledPlan ) {
+			// "a" requires an annually billed plan, "b" does not
+			return -1;
+		}
+
+		// By exclusion, "b" requires an annually billed plan, "a" does not
+		return 1;
+	} );
+
+	return features;
+}
```

Comment:
It might be less readable, but this is the same as: 

```suggestion
    // Features requiring an annually billed plan should be first in the array.
	features.sort(
		( a, b ) => Number( a.requiresAnnuallyBilledPlan ) - Number( b.requiresAnnuallyBilledPlan )
	);
```

Reply:
I personally prefer code that is easy to understand, but I quite like how neat this looks!

Reply:
Turns out it had to be the other way around (`b - a`)

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/48963/commits/db937b810c139622c334991288de4967ccbd6073